packages/audioplayers/tizen/src/audio_player.cc

```diff
@@ -164,8 +176,15 @@ void AudioPlayer::SetDataSource(std::vector<uint8_t> &data) {
 }
 
 void AudioPlayer::SetVolume(double volume) {
+  if (volume > 1) {
+    volume = 1;
+  } else if (volume < 0) {
+    volume = 0;
+  }
   volume_ = volume;
-  if (GetPlayerState() != PLAYER_STATE_NONE) {
+
+  player_state_e state = GetPlayerState();
+  if (state != PLAYER_STATE_NONE) {
```

Comment:
This `state` won't be reused so, i don't think it's necessary.
+) 87,220 lines are  also.